Mathlib/Algebra/Group/Defs.lean

```diff
@@ -752,14 +771,8 @@ instance (priority := 100) CancelCommMonoid.toCancelMonoid (M : Type u) [CancelC
 #align cancel_comm_monoid.to_cancel_monoid CancelCommMonoid.toCancelMonoid
 #align add_cancel_comm_monoid.to_cancel_add_monoid AddCancelCommMonoid.toAddCancelMonoid
 
-/-- Any `CancelMonoid G` satisfies `IsCancelMul G`. -/
-@[to_additive toIsCancelAdd "Any `AddCancelMonoid G` satisfies `IsCancelAdd G`."]
-instance (priority := 100) CancelMonoid.toIsCancelMul (M : Type u) [CancelMonoid M] :
-    IsCancelMul M :=
-  { mul_left_cancel := LeftCancelSemigroup.mul_left_cancel
-    mul_right_cancel := RightCancelSemigroup.mul_right_cancel }
-#align cancel_monoid.to_is_cancel_mul CancelMonoid.toIsCancelMul
-#align add_cancel_monoid.to_is_cancel_add AddCancelMonoid.toIsCancelAdd
+#noalign cancel_monoid.to_is_cancel_mul
+#noalign add_cancel_monoid.to_is_cancel_add
```

Comment:
I think these should be aligned with the semigroup versions, to the caller (using only explicit arguments) they are equivalent.